pandas/core/series.py

```diff
@@ -173,12 +174,16 @@ def __init__(self, data=None, index=None, dtype=None, name=None,
                 raise NotImplementedError("initializing a Series from a "
                                           "MultiIndex is not supported")
             elif isinstance(data, Index):
-                # need to copy to avoid aliasing issues
                 if name is None:
                     name = data.name
 
-                data = data._to_embed(keep_tz=True, dtype=dtype)
```

Comment:
Is there a reason the `keep_tz=True` is no longer needed? (or was this never used) 
From looking at the `_to_embed` code it seems the difference is a deep=True in copy and the fact it returned a DatetimeIndex instead of array.

Reply:
Ah, I suppose that is now what `_values` does? (the distinction in return type)